store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStore.java

```diff
@@ -124,8 +126,12 @@
     private Connector connection = null;
 
     @Override
-    public void initialise(final Schema schema, final StoreProperties properties)
-            throws StoreException {
+    protected Class<? extends Serialiser> getRequiredParentSerialiserClass() {
```

Comment:
Sorry to be picky but please could this method go further down the class with the other protected methods. It is nice to have the initialise method first. Similarly in the other store implementations.

Also you could change the return type to Class<? extends ToBytesSerialiser>